scripts/globals/crafting.lua

```diff
@@ -204,6 +204,10 @@ xi.crafting.getCraftSkillCap = function(player, craftID)
     return (rank + 1) * 10
 end
 
+xi.crafting.getRealSkill = function(player, craftID)
+    return math.floor(player:getCharSkillLevel(xi.skill.CLOTHCRAFT) / 10)
```

Comment:
Shouldn't `player:getCharSkillLevel(xi.skill.CLOTHCRAFT)` be `player:getCharSkillLevel(craftID)`?

Reply:
Bad copy-pasta, fixing.